terraform/modules/api-gateway/api_gateway_lambda.js

```diff
@@ -1,11 +1,69 @@
+const AWS = require('aws-sdk');
+const jwt = require("jsonwebtoken");
+const jwksClient = require("jwks-rsa");
+
+const secretsManager = new AWS.SecretsManager();
+let cachedSecrets = null;
+
+async function getSecrets() {
+    if (cachedSecrets) return cachedSecrets;
```

Comment:
Might be worth commenting that this will only cache for the duration in which that particular lambda invocation is warm for

Reply:
Added